src/flask_cognito_lib/services/token_svc.py

```diff
@@ -188,3 +194,33 @@ def verify_id_token(
             raise TokenVerifyError("Token nonce check failed")
 
         return claims
+
+    def encrypt_token(self, token: str) -> str:
+        """Encrypt a token using the configured key
+
+        Parameters
+        ----------
+        token : str
+            The token to encrypt
+
+        Returns
+        -------
+        str
+            The encrypted token
+        """
+        return self.fernet.encrypt(token.encode()).decode()
+
+    def decrypt_token(self, token: str) -> str:
+        """Decrypt a token using the configured key
+
+        Parameters
+        ----------
+        token : str
+            The token to decrypt
+
+        Returns
+        -------
+        str
+            The decrypted token
+        """
+        return self.fernet.decrypt(token.encode()).decode()
```

Comment:
Should we have error handling here for failed encode/decode, and failed encrypt/decrpyt?